definitions/ext-aws_elemental/summary_metrics.yml

```diff
@@ -0,0 +1,27 @@
+ipAddress:
```

Comment:
The same comment from the [previous PR](https://github.com/newrelic/entity-definitions/pull/742/files#r962804959) applies here too. 

Sorry about the delay and thanks in advance!